updateCertificationStatusApp.sh

```diff
@@ -5,4 +5,20 @@
 # 15 5 * * * cd /tempApps && ./updateCertificationStatusApp.sh
 # This will run it at 0515 UTC, which (depending on DST) is 0015 EST
 
-java -jar certStatusApp-jar-with-dependencies.jar
\ No newline at end of file
+# create timestamp and filename
+TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+log=logs/log.certStatusApp.$TIMESTAMP.txt
```

Comment:
It does exist on my local machine and on the Jenkins machine where this is deployed. However, for some reason git isn't noticing the added folder and reflecting it when I run "git status" even after doing a "git fetch origin".